packages/workspace/src/tasks-runner/cache.ts

```diff
@@ -106,10 +100,10 @@ export class Cache {
       await Promise.all(
         outputs.map(async (f) => {
           const src = join(this.root, f);
-          if (await existsAsync(src)) {
+          if (existsSync(src)) {
```

Comment:
Why did this change to sync? I know it's pretty fast to check if things exist but it should stay async. Please change all of the other places as well.

Reply:
@FrozenPandaz async exists is deprecated and should not be used according to the Node.js Documentation. https://nodejs.org/api/fs.html#fsexistspath-callback